detox.config.js

```diff
@@ -0,0 +1,77 @@
+/** @type {Detox.DetoxConfig} */
+module.exports = {
+  logger: {
+    level: process.env.CI ? 'debug' : undefined,
+  },
+  testRunner: {
+    args: {
+      config: 'e2e/jest.config.js',
+      maxWorkers: 1,
+    },
+  },
+  artifacts: {
+    plugins: {
+      log: process.env.CI ? 'failing' : undefined,
+      screenshot: process.env.CI ? 'failing' : undefined,
```

Comment:
Don't we want to take screenshots when tests fail locally? This worked differently before.

Reply:
Agreed this should always be "failing"

Reply:
```suggestion
      screenshot: 'failing',
```